static/app/components/events/interfaces/debugMeta/debugImageDetails/candidates.tsx

```diff
@@ -138,7 +135,7 @@ class Candidates extends Component<Props, State> {
     const filterOptions = this.getFilterOptions(candidates);
 
     const defaultFilterSelections = (
-      filterOptions.find(section => section.value === 'status')?.options ?? []
+      filterOptions.find(section => section.key === 'status')?.options ?? []
```

Comment:
Select sections no longer accept a `value` prop (they are not inherently selectable, so it doesn't make sense for them to have a select `value`). To distinguish between different sections, we can use the optional `key` prop instead.